examples/tutorials/vectors.py

```diff
@@ -245,32 +245,25 @@
 # # Plot Circular Vectors
 # ----------
 #
-# Circular vectors can be plotted using an ``x`` and ``y`` value to specify
-# where the origin of the circle will be located on the plane. The variable
-# ``diam`` is used to specify the diameter of the circle while the ``startDeg``
-# and ``stopDeg`` specify at what angle the arc will begin and end respectively.
+# Circular vectors can be plotted using an ``x_start`` and ``y_start`` value to
+# specify where the origin of the circle will be located on the plane. The
+# variable ``radius`` is used to specify the radius of the circle while the
+# ``degree_start`` and ``degree_stop`` parameters specify at what angle the arc
+# will begin and end respectively.
 
-fig = pygmt.Figure()
-
-reg_x_lowbound = 0
-reg_x_upperbound = 8
-reg_y_lowbound = -15
-reg_y_upperbound = 15
+# vector specifications structured as: [x_start, y_start, radius, degree_start, degree_stop]
+data = np.array([[4, 4, 2, 90, 270]])
```

Comment:
I think this should be split up into two steps. One for the creation of the simple list and the other for the numpy array

```suggestion
vector_1 = [4, 4, 2, 90, 270]
data = np.array([vector_1])
```